crates/symbolicator/src/services/mod.rs

```diff
@@ -91,6 +91,7 @@ impl Service {
             symcaches,
             cficaches,
             caches.diagnostics,
+            io_pool,
```

Comment:
afaik this pool is only constructed with 4 threads at the moment because it is only used by the download service until now.  We should now construct it with the default number of threads I think, which is the number of available CPUs afaik.